### PR TITLE
Better history functions & fix bug with sync add_entity call

### PR DESCRIPTION
**Pull request by kizill**
I'm trying to play with history functions and have found that *args & **kwargs are not the best friends of mine :)
Also fixed small bug with `add_entity` call.

### CODE REVIEW

File: homeassistant_api/_async/asyncclient.py [hunk 1]
```diff
@@ -226,7 +237,7 @@ async def async_get_state(  # pylint: disable=duplicate-code
             slug=slug,
             entity_id=entity_id,
         )
-        data = await self.async_request(join("states", target_entity_id))
+        data = await self.async_request(join("states/", target_entity_id))
```
Comment (GrandMoff100) [inline homeassistant_api/_async/asyncclient.py]: Is the trailing slash necessary? Just curious.

File: homeassistant_api/models/entity.py [hunk 2]
```diff
@@ -25,7 +26,7 @@ def add_entity(self, entity_slug: str, state: State) -> None:
         self.entities[entity_slug] = Entity(
             slug=entity_slug,
             state=state,
-            client=self.client,
+            group=self,
```
Comment (GrandMoff100) [inline homeassistant_api/models/entity.py:29]: Wow! How did I miss this! Thank you for finding this.

**Comment by GrandMoff100** [comment]
Nice! Your PR is very welcome 🙏. I think removing *arts and **kwargs will save future devs some headaches lol.

I have a few questions that I'm curious about and then we can merge it. Thoughts?

**Comment by kizill** [comment]
Hello!
What should i do now? :)

**Comment by kizill** [comment]
Friendly ping :)

**Comment by GrandMoff100** [comment]
In my review I had some questions on a few small things. After we talk about those then I can merge it!

**Comment by GrandMoff100** [comment]
Ah there we go.

File: homeassistant_api/_async/asyncclient.py (hunk 1 above)
Comment (kizill) [reply homeassistant_api/_async/asyncclient.py]: Nope, just artifact from first "hotfix" for #101  :)

Comment (GrandMoff100) [reply homeassistant_api/_async/asyncclient.py]: Awesome, I'll probably remove these sooner or later then. 👍

**Review by GrandMoff100 — APPROVED** [review]

**Comment by GrandMoff100** [comment]
Here we go!

**Pull request merged**